tests/test_msi_workflow_cwl.py

```diff
@@ -0,0 +1,296 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Test case for the MSI analysis workflow cwl which uses multiple input samples
+"""
+import os
+import sys
+import unittest
+
+THIS_DIR = os.path.dirname(os.path.realpath(__file__))
+PARENT_DIR = os.path.dirname(THIS_DIR)
+sys.path.insert(0, PARENT_DIR)
+from pluto.tools import PlutoTestCase, CWLFile, TableReader
+sys.path.pop(0)
+
+class TestMsiWorkflow(PlutoTestCase):
+    cwl_file = CWLFile('msi_workflow.cwl')
+
+    def setUp(self):
+        # initialize the tmpdir
+        super().setUp()
+        self.data_clinical_lines = [
+        ['#SAMPLE_ID', 'PATIENT_ID', 'SAMPLE_COVERAGE'],
+        ['#SAMPLE_ID', 'PATIENT_ID', 'SAMPLE_COVERAGE'],
+        ['#STRING', 'STRING', 'NUMBER',],
+        ['#1', '1', '1'],
+        ['SAMPLE_ID', 'PATIENT_ID', 'SAMPLE_COVERAGE'],
+        ['Sample1-T', 'Patient1', '108'],
+        ['Sample1-N', 'Patient2', '58'],
+        ['Sample2-T', 'Patient3', '502'],
+        ['Sample2-N', 'Patient4', '56'],
+        ['Sample6-T', 'Patient4', '57'],
+        ['Sample7-N', 'Patient4', '58'],
+        ]
+
+        self.data_clinical_file = self.write_table(self.tmpdir, filename = "data_clinical_sample.txt", lines = self.data_clinical_lines)
+        self.normal_bam = "/work/ci/vurals/msi_test/test_stuff/foo_normal.rg.md.bam"
```

Comment:
There are already some bam files we can use for testing, I think they should be referenced under the test case for `snp-pileup`.